cuequivariance/cuequivariance/descriptors/irreps_tp.py

```diff
@@ -80,6 +80,62 @@ def fully_connected_tensor_product(
     )
 
 
+def full_tensor_product(
+    irreps1: cue.Irreps,
+    irreps2: cue.Irreps,
+    irreps3_filter: Optional[Sequence[cue.Irrep]] = None,
+) -> cue.EquivariantTensorProduct:
+    """
+    subscripts: ``lhs[ui],rhs[vj],output[uvk]``
```

Comment:
Hi Mit, 

Thanks for the contribution!
Here the subscripts are different than in the output descriptor.
The rest of the changes look good to me.

Reply:
Thanks for the review ! Resolved